### ISSUE TITLE
gui_qml build is broken

### ISSUE THREAD

**Issue by mdankov** [navit-gps/navit#942]
@metalstrolch,  commit https://github.com/navit-gps/navit/commit/10f514af97b3491072a172b3e0f3e78de455bfb3 seems to break gui_qml build:
```
In file included from /work/compile/navit/src/navit/gui/qml/gui_qml.cpp:89:0:
/work/compile/navit/src/navit/gui/qml/navitProxy.h: In member function virtual attr_iter* NGQProxyNavit::getIterFunc():
/work/compile/navit/src/navit/gui/qml/navitProxy.h:141:63: error: too few arguments to function attr_iter* navit_attr_iter_new(void*)
  struct attr_iter* getIterFunc() { return navit_attr_iter_new(); };
                                                               ^
In file included from /work/compile/navit/src/navit/gui/qml/gui_qml.cpp:18:0:
/work/compile/navit/src/navit/navit.h:113:19: note: declared here
 struct attr_iter *navit_attr_iter_new(void * unused);
                   ^~~~~~~~~~~~~~~~~~~
make[2]: *** [navit/gui/qml/CMakeFiles/gui_qml.dir/build.make:108: navit/gui/qml/CMakeFiles/gui_qml.dir/gui_qml.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:589: navit/gui/qml/CMakeFiles/gui_qml.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

**Comment by metalstrolch** [comment]
Interestingly it doesn't on CI. I'll have a look on it.

**Comment by metalstrolch** [comment]
Should be already fixed in #940 once that's merged. @pgrandin was quite fast in merging #935. I wasn't finished with that.

**Comment by mdankov** [comment]
#940 seems to be merged now, and I see the change that should fix the issue.
I have already worked it around on the map server by excluding gui_qml build. Actually the only thing I need is maptool.
Feel free to close it unless you want to ..
> Interestingly it doesn't on CI. I'll have a look on it.

Actually gui qml build is bypassed here: https://github.com/navit-gps/navit/blob/87dabb3edcac268a49f2cb5da626abcb751b47bb/scripts/build_linux.sh#L7

**Comment by metalstrolch** [comment]
Ah OK. I mixed it up with qt5_qml. Wondering if the qt4 graphics and qml gui are still used by anybody at all. At least they are unmaintained for years. Closing this as it is fixed with #940.
